ReleaseProcessScript/tools/Core/jira_functions.ps1

```diff
@@ -84,7 +84,14 @@ function Jira-Get-Current-Version ()
 
   Add-Authentication $JiraGetVersion
 
-  $JiraGetVersion.Execute()
+  try
+  {
+    $JiraGetVersion.Execute()
+  }
+  catch
+  {
+    throw $_.Exception.Message
```

Comment:
I.C.

Probably should add a code-comment for these two blocks because it's really not that obvious without knowning the history.

Reply:
Understandable. However, I did not do so because these `try-catch` blocks have been in every function from `jira_functions.ps1` (except for the two methods affected by the changes in this PR). Do you want me to add a comment to each `try-catch` block or e. g. a comment at the top of the file that addresses every jira function?